data_model.mdd

```diff
@@ -262,10 +264,22 @@
         int revised_outcome FK "Review of Progression Revised Outcome, Null: True"
         string(MAX) comments "Review of Progression Comments, Null: True"
     }
+    MidReviewProgression {
```

Comment:
different name to the relationship part 
`TrainingRecord ||--o{ MidtermReviewProgression: has`

Reply:
Fixed this along with some other typos